content/riak/ts/1.3.0/using/configuring.md

```diff
@@ -36,22 +36,23 @@ timeseries_query_timeout_ms = 10000
 
 ### Maximum quanta
 
-Use `timeseries_query_max_quanta_span` to configure the maximum number of quanta that a query can span
+Use `timeseries_query_max_quanta_span` to configure the maximum number of quanta that a query can span. The default is 5.
 
 ```
-timeseries_query_max_quanta_span = 3
+timeseries_query_max_quanta_span = 5
 ```
 
-For example, if you set `QUANTUM(time, 15, 'm')` in your TS table, setting `timeseries_query_max_quanta_span` to '5' would allow a query to return results within a time span of 75 minutes.  If a query has a larger time span, an error would be returned to the client and the query would not run.
+You should set this parameter to be one more than your desired timespan, else a fraction of your queries will fall outside your `timeseries_query_max_quanta_span` window. In general, your timespan won't be an integer multiple of the quantum, so if your maximum size query is dtmax, and your quantum size is Q, then we recommend: 
+max_quanta = ceil(dtmax / Q + 1.0). 
 
-This option is intended to prevent excessively long-running queries that could affect the performance of the cluster.
+If a query has a larger time span than your specified maximum, the error `too_many_subqueries` will show and the query will not run. This option is intended to prevent excessively long-running queries that could affect the performance of the cluster.
```

Comment:
@TheOnlyRew It has to do with if you try to query longer than your parameterized time span, you'll get an error.